backend/commitments/models.py

```diff
@@ -0,0 +1,14 @@
+from django_cassandra_engine.models import DjangoCassandraModel
+from cassandra.cqlengine import columns
+from uuid import uuid4
+
+class CommitmentPacts(DjangoCassandraModel):
+    id = columns.UUID(primary_key=True, default=uuid4)
+    id_client = columns.UUID(required=True)
+    creation_date = columns.DateTime(required=True)  # Date of the creation of the "Pacte d'engagement"
+    answers_commitments = columns.List(columns.UUID(required=True), required=True) # List of answers with isEngagement = True
+    calculated_score = columns.Double(required=True)  # Calculated score of the module in validation state
+
+    @classmethod
+    def get_by_id(cls, id_commitment_pact):
+        return CommitmentPacts.objects.get(id=id_commitment_pact)
```

Comment:
Did you deleted the class CommitmentPacts in the "module" app ?

Reply:
With the git pull origin main it duplicates the models thanks i have resolve the issue